fission/src/test/MirabufParser.test.ts

```diff
@@ -2,12 +2,11 @@ import { describe, test, expect } from "vitest"
 
 import { mirabuf } from "../proto/mirabuf"
 import MirabufParser, { RigidNodeReadOnly } from "../mirabuf/MirabufParser"
-import { LoadMirabufRemote } from "@/mirabuf/MirabufLoader"
-// import { LoadMirabufLocal } from "../mirabuf/MirabufLoader"
+import { LoadMirabufRemote, MiraType } from "../mirabuf/MirabufLoader"
```

Comment:
Reason for this not being imported like it was previously with `@`? Not sure why these are all over the place right now but this is not the only case so perhaps there is something that I am missing?

Reply:
I prefer using the @ path aliases because it means if we were to refactor and move mirabuf stuff somewhere else (or something like that) we would only have to change where the path alias points to rather than changing all the imports.